OrbitGl/App.cpp

```diff
@@ -785,6 +781,17 @@ bool OrbitApp::SelectProcess(const std::string& process) {
   return false;
 }
 
+bool OrbitApp::IsCaptureConnected(const CaptureData& capture) const {
+  const ProcessData* selected_process = GetSelectedProcess();
+  if (selected_process == nullptr) return false;
+
+  const ProcessData* capture_process = capture.process();
+  CHECK(capture_process != nullptr);
+
+  return selected_process->pid() == capture_process->pid() &&
+         selected_process->full_path() == capture_process->full_path();
```

Comment:
TODO: check buildid?

Reply:
I put a TODO and it's a good idea to think about the details of what that would actually mean. Are build-ids required and  so on.